packages/api/src/graphql/mutation.private.ts

```diff
@@ -412,9 +412,11 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
         input: {type: GraphQLNonNull(GraphQLUserInput)},
         password: {type: GraphQLNonNull(GraphQLString)}
       },
-      resolve(root, {input, password}, {authenticate, dbAdapter}) {
+      async resolve(root, {input, password}, {authenticate, dbAdapter}) {
         const {roles} = authenticate()
         authorise(CanCreateUser, roles)
+        input.email = input.email.toLowerCase()
```

Comment:
Imo we should implement that in the database package to tackle it at the root. Whenever another person will implement an additional user endpoint which includes somehow the email, he or she will not know, that she or he has to implement email.toLowerCase()
This is maintainability imo.

Reply:
That was how i implemented it first and Karl told me then he thinks thats wrong so i changed this. So we leaf it for now link this ok? ;-)

Reply:
ok. but want to state, that this is less maintainable.

Reply:
We should fix it on the API, if we fix it on the database layer then everything before is not using the correct data. If  we fix it on the API, everything is using the correct data.

My suggestion was to create a ScalarType for email to validate it, that's the recommended way by Apollo too.

Reply:
Ok. Good argument. Agree.